pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/scheduler/TransferShedder.java

```diff
@@ -444,17 +444,21 @@ private boolean isTransferable(LoadManagerContext context,
         if (pulsar == null || allocationPolicies == null) {
             return true;
         }
+        var isLoadBalancerSheddingBundlesWithPoliciesEnabled =
+                context.brokerConfiguration().isLoadBalancerSheddingBundlesWithPoliciesEnabled();
         String namespace = LoadManagerShared.getNamespaceNameFromBundleName(bundle);
         final String bundleRange = LoadManagerShared.getBundleRangeFromBundleName(bundle);
         NamespaceBundle namespaceBundle =
                 pulsar.getNamespaceService().getNamespaceBundleFactory().getBundle(namespace, bundleRange);
 
-        if (!canTransferWithIsolationPoliciesToBroker(
+        if (!isLoadBalancerSheddingBundlesWithPoliciesEnabled
```

Comment:
I'm a little confused. When getting `TopKBundles` if `isLoadBalancerSheddingBundlesWithPoliciesEnabled` is false and the bundle has an affinity(anti) policy, the bundle will be discarded.

Reply:
The above table is correct. Thank you for this catch. Updated the code.